aws_sagemaker_studio/sagemaker_data_prep_assistant/data_prep_widget.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>Maybe a 102 sentence summery on what the interactive data preparation assistant widget and how it can help the customer?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>